<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkLogWindow</class>
 <widget class="QWidget" name="WorkLogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work Log</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/log_window.png</normaloff>../images/log_window.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #f0f0f0;
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 5px 10px;
    min-width: 70px;
}

QPushButton:hover {
    background-color: #e0e0e0;
    border: 1px solid #aaa;
}

QPushButton:pressed {
    background-color: #d0d0d0;
}

QPushButton:disabled {
    background-color: #f5f5f5;
    color: #999;
}

QLineEdit, QTextEdit, QPlainTextEdit {
    border: 1px solid #ccc;
    border-radius: 3px;
    padding: 3px;
}

QComboBox {
    border: 1px solid #ccc;
    border-radius: 3px;
    padding: 3px;
}

QCheckBox, QRadioButton {
    spacing: 5px;
}

QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px;
}

QScrollBar:vertical {
    border: none;
    background: #f5f5f5;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #d0d0d0;
    min-height: 20px;
    border-radius: 5px;
}

QTabWidget::pane {
    border: 1px solid #ccc;
    border-radius: 4px;
}

QTabBar::tab {
    background: #f0f0f0;
    border: 1px solid #ccc;
    border-bottom: none;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    padding: 5px 10px;
}

QTabBar::tab:selected {
    background: white;
    margin-bottom: -1px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="workLogPlainTextEdit">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="horizontalGroupBox">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <widget class="QPushButton" name="copyButton">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
